test/extended/networking/livemigration.go

```diff
@@ -33,6 +33,8 @@ import (
 var _ = Describe("[sig-network][OCPFeatureGate:PersistentIPsForVirtualization][Feature:Layer2LiveMigration] Kubevirt Virtual Machines", func() {
 	oc := exutil.NewCLIWithPodSecurityLevel("network-segmentation-e2e", admissionapi.LevelBaseline)
 	f := oc.KubeFramework()
+	// disable automatic namespace creation, we need to add the required UDN label
+	f.SkipNamespaceCreation = true
```

Comment:
With your changes there will be two namespaces created, the one you added and the one created by:
`oc := exutil.NewCLIWithPodSecurityLevel("network-segmentation-e2e", admissionapi.LevelBaseline)`

We should probably use `exutil.NewCLIWithoutNamespace` instead.

Reply:
done